api_v3/services/LiveStreamService.php

```diff
@@ -617,6 +618,46 @@ public function archiveAction($liveEntryId, $vodEntryId)
         return true;
     }
 
+	protected function responseHandlingGetStats($isLive)
+	{
+		$liveStatsInterval = kConf::get('liveStatsInterval','live');
+		KalturaResponseCacher::setExpiry($liveStatsInterval);
+		KalturaResponseCacher::setHeadersCacheExpiry($liveStatsInterval);
+	}
+
+	/**
+	 * Deliver information about the livestream
+	 *
+	 * @action getLiveStreamStats
+	 * @param string $entryId Id of the live stream entry
+	 * @return KalturaLiveStreamStats
+	 * @ksIgnored
+	 *
+	 * @throws KalturaErrors::INVALID_ENTRY_ID
+	 */
+	public function getLiveStreamStatsAction($entryId)
+	{
+		$liveStreamEntry = $this->fetchLiveEntry($entryId);
+		$isLive = $liveStreamEntry->isCurrentlyLive();
+		$liveStreamInfo = new KalturaLiveStreamStats();
+		$liveStreamInfo->liveViewers = $isLive ? $this->getNumberOfViewers($entryId) : 0;
```

Comment:
lets avoid extracting the entry and just return what we have in cache, if its not live the value will be null